packages/nextjs/src/common/wrapServerComponentWithSentry.ts

```diff
@@ -81,6 +82,7 @@ export function wrapServerComponentWithSentry<F extends (...args: any[]) => any>
           maybePromiseResult = originalFunction.apply(thisArg, args);
         } catch (e) {
           handleErrorCase(e);
+          void flush();
```

Comment:
didn't we talk about this a while ago and we said we need to await the flush call?

Reply:
Yes, we talked about this. Here we are in CF worker land where execution freeze should not be an issue.

Reply:
Perfect, thx!